balancer-js/src/modules/pools/pool-types/concerns/metaStable/exit.concern.integration.spec.ts

```diff
@@ -243,10 +245,13 @@ describe('exit meta stable pools execution', async () => {
 
       it('tokens balance should increase by exact amountsOut', async () => {
         for (let i = 0; i < tokensBalanceAfter.length; i++) {
+          // MetaStable exitExactTokensOut may exit with a rounding error of 1 wei
           expect(
             tokensBalanceAfter[i]
               .sub(tokensBalanceBefore[i])
-              .eq(tokensMinBalanceIncrease[i])
+              .sub(tokensMinBalanceIncrease[i])
+              .abs()
+              .lte(1)
```

Comment:
Was this required because the tests started to fail?

Reply:
This is actually a hack that I copied over from the frontend. 
It was previously being handled within the concern ([here](https://github.com/balancer-labs/balancer-sdk/blob/682fe7dc5346e69bb47be1771e7ecf38f8e46711/balancer-js/src/modules/pools/pool-types/concerns/metaStable/exit.concern.ts#L266)).
Now that I'm not returning `minAmountsOut` for `exitExactTokensOut`, I had to move it to the test.
Let me know if you think there's a better way to handle it.

Reply:
How about using [clostTo](https://www.chaijs.com/api/bdd/#method_closeto)?